### DISCUSSION TITLE
Theta theme: set number of MRU items?

### DISCUSSION THREAD

**Discussion by liouk** [goolord/alpha-nvim#159] [Q&A] [answered]
Hello!

I am using the theta theme (awesome btw), and I'm trying to figure out how to set the number of items to be shown in the MRU file list. I tried re-defining the section and calling `mru(0, cdir, 3)` but I can't seem to be able to access the `mru()` function from outside the module (expected as it's declared local -- my lua knowledge is very limited!).

Is there any way to do this, without redefining the whole theme?

Thanks!

**Comment by goolord** [comment, marked as answer]
`require('alpha.themes.startify').mru(0, cdir, 3)`

**Reply by liouk** [reply]
Thank you!! It never occurred to me to check startify 😅

**Reply by liouk** [reply]
FYI I've had a look, and it seems that the startify MRU function is quite different than the theta one, i.e. button structure is different and also plenary isn't called on startify to shorten the path.

Maybe exporting the theta's MRU would also help in this case! I'll try to see if I can get it to work that way. Thanks again!

**Reply by liouk** [reply]
Created a PR for this 👉 https://github.com/goolord/alpha-nvim/pull/160

With the PR, this could be done as shown below:
```lua
local config = require("alpha.themes.theta").config
local mru_files = config.layout[4]

mru_files.val[3] = {
  type = "group",
  val = function()
    return { require("alpha.themes.theta").mru(0, cdir, 3) }
  end,
  opts = { shrink_margin = false },
}
```

**Comment by goolord** [comment]
it's "exported" here https://github.com/goolord/alpha-nvim/blob/main/lua/alpha/themes/startify.lua#L245

**Comment by gennaro-tedesco** [comment]
Could it be useful perhaps to add proper documentation for `theta` too, as a theme? At the moment it's quite confusing to have to search in the GitHub issues to understand how the theta API works (the grammar seems to be inconsistent across all themes)?

**Reply by goolord** [reply]
i have tried to keep the `dashbaord` / `startify` apis mostly the same. `theta` is not documented precisely because i haven't yet defined an api for it that i can promise won't break. if you don't like it just copy the module yourself, alpha themes are expressed only as data for this exact reason. 

or better yet improve the code and documentation yourself.
this plugin already serves its full utility for me and i make changes to it with my own free time, so if you wait for me to make a change for you you're just going to have to be patient :)
